kinsumer.go

```diff
@@ -234,6 +234,42 @@ func (k *Kinsumer) dynamoTableActive(name string) error {
 	return nil
 }
 
+// dynamoTableExists returns an true if the given table exists
+func (k *Kinsumer) dynamoTableExists(name string) bool {
+	_, err := k.dynamodb.DescribeTable(&dynamodb.DescribeTableInput{
+		TableName: aws.String(name),
+	})
+	return err == nil
+}
+
+// dynamoCreateTable creates a table with the given name and distKey
+func (k *Kinsumer) dynamoCreateTable(name, distKey string) error {
```

Comment:
aws-sdk-go has `waiters` (https://github.com/aws/aws-sdk-go/blob/master/service/dynamodb/waiters.go) that are a reliable way to know if the create worked or failed. Thoughts on adding these waiters to `dynamoCreateTable` and `dynamoDeleteTable` Alternatively to `CreateRequiredTables` ?